migrations/versions/49a07c84feee_create_vector_db_table.py

```diff
@@ -0,0 +1,76 @@
+"""create vector_db table
+
+Revision ID: 49a07c84feee
+Revises: ba60b12ae109
+Create Date: 2023-07-10 09:12:30.989026
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '49a07c84feee'
+down_revision = 'ba60b12ae109'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('knowledge',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('name', sa.String(), nullable=True),
+    sa.Column('description', sa.String(), nullable=True),
+    sa.Column('summary', sa.String(), nullable=True),
+    sa.Column('readme', sa.String(), nullable=True),
+    sa.Column('index_id', sa.Integer(), nullable=True),
+    sa.Column('organisation_id', sa.Integer(), nullable=True),
+    sa.Column('contributed_by', sa.String(), nullable=True),
+    sa.Column('created_at', sa.DateTime(), nullable=True),
+    sa.Column('updated_at', sa.DateTime(), nullable=True),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('marketplace_state',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('reference_id', sa.Integer(), nullable=True),
+    sa.Column('reference_name', sa.String(), nullable=True),
+    sa.Column('key', sa.String(), nullable=True),
+    sa.Column('value', sa.Integer(), nullable=True),
+    sa.Column('created_at', sa.DateTime(), nullable=True),
+    sa.Column('updated_at', sa.DateTime(), nullable=True),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('vector_db',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('name', sa.String(), nullable=True),
+    sa.Column('db_type', sa.String(), nullable=True),
+    sa.Column('organisation_id', sa.Integer(), nullable=True),
+    sa.Column('created_at', sa.DateTime(), nullable=True),
+    sa.Column('updated_at', sa.DateTime(), nullable=True),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('vector_db_index_collection',
```

Comment:
should be vector_db_indexes. index_collection does not make sense.

name, vector_db_id can't be nullable